From 06b1a19d7cb309200b6f86f0bc0d322d0bf7006f Mon Sep 17 00:00:00 2001
From: David Daney <ddaney@caviumnetworks.com>
Date: Thu, 23 Jun 2011 14:58:04 -0700
Subject: [PATCH 69/97] netdev: octeon-ethernet: Allow receive processing to be limited to one CPU.

Source: Cavium SDK 2.1.0-407

A typo caused the minimum to be two CPUs.

Signed-off-by: David Daney <ddaney@caviumnetworks.com>
Integrated-by: Phil Staub <Phil.Staub@windriver.com>
---
 drivers/net/octeon/ethernet-rx.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/octeon/ethernet-rx.c b/drivers/net/octeon/ethernet-rx.c
index 6876e22..1b8ac8a 100644
--- a/drivers/net/octeon/ethernet-rx.c
+++ b/drivers/net/octeon/ethernet-rx.c
@@ -359,7 +359,7 @@ void cvm_oct_rx_initialize(int num_wqe)
 					struct octeon_ethernet,
 					list)->netdev;
 
-	if (max_rx_cpus > 1  && max_rx_cpus < num_online_cpus())
+	if (max_rx_cpus >= 1  && max_rx_cpus < num_online_cpus())
 		core_state.baseline_cores = max_rx_cpus;
 	else
 		core_state.baseline_cores = num_online_cpus();
-- 
1.7.0

